libnavigation-core/src/test/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetryTest.kt

```diff
@@ -540,167 +547,99 @@ class MapboxNavigationTelemetryTest {
     }
 
     @Test
-    fun cache_feedback_is_cached() {
```

Comment:
Are we ok removing these cache feedback tests? 👇

Reply:
`cache feedback` is not presented anymore, now only FeedbackMetadata exists 🚀

Reply:
This also applies to other tests removed below 👇
Want to make sure we're not removing any useful tests.